File: PizzaDelivery/DeliveryApp/src/app/models/store.ts
import { Inventory } from "./inventory";
import { Order } from "./order";

export class Store {
    id?: number;
    storeNumber: string;
    phone: string;
    address: string;
    city: string;
    state: string;
    zip: string;
    inventory: Inventory[];
    orders: Order[];
}

File: PizzaDelivery/PizzaDelivery/Models/Store.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace PizzaDelivery.Models
{
    public class Store
    {
        public int Id { get; set; }

        public string StoreNumber { get; set; }
        [Required]
        public string Phone { get; set; }
        [Required]
        public string Address { get; set; }
        [Required]
        public string City { get; set; }
        [Required]
        public string State { get; set; }
        [Required]
        public string Zip { get; set; }
        public List<Inventory> Inventory { get; set; }
        public List<Order> Orders { get; set; }
    }
}
